ui/src/router/components/request_field_source/FieldSourceFormLabel.js

```diff
@@ -9,23 +9,36 @@ import {
 
 import "./FieldSourceFormLabel.scss";
 
-const fieldSourceOptions = [
-  {
-    value: "header",
-    inputDisplay: "Header",
-  },
-  {
-    value: "payload",
-    inputDisplay: "Payload",
-  },
-];
+const fieldSourceOptions = {
```

Comment:
<img width="644" alt="Screenshot 2022-10-27 at 9 37 58 AM" src="https://user-images.githubusercontent.com/23465343/198170951-b2ae7ce4-92f1-4181-b86a-780778e19303.png">

I encountered this problem when attempting to use an internal standard experiment engine with UPI.

Reply:
added a filter in experiment step `ui/src/router/components/form/steps/ExperimentStep.js` as UPI doesn't intend to support standard experiment engine yet.